MAiD - Case Management App/force-app/main/default/permissionsets/MAiD_Analyst.permissionset-meta.xml

```diff
@@ -10,8 +10,13 @@
         <readable>true</readable>
```

Comment:
These fields were found to be absent for MAiD teams afer View All permission was removed.